support-frontend/assets/pages/digital-subscription-landing/components/australianEditions/productBlockAus.jsx

```diff
@@ -0,0 +1,249 @@
+/* eslint-disable react/no-unused-prop-types */
+// @flow
+import React, { Component, type Node } from 'react';
+import AdFreeSection from 'components/adFreeSection/adFreeSection';
+import { trackComponentClick } from 'helpers/tracking/behaviour';
+import GridPicture from 'components/gridPicture/gridPicture';
+import cx from 'classnames';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { ListHeading } from 'components/productPage/productPageList/productPageList';
+
+// styles
+import '../digitalSubscriptionLanding.scss';
+
+const Plus = () => <div className="product-block__plus">+ Plus</div>;
+
+const arrowSvg = (
+  <svg width="18" height="10" xmlns="http://www.w3.org/2000/svg">
+    <defs><path d="M16 0l1.427 1.428-7.035 7.036.035.035L9 9.927l-.035-.035-.036.035L7.5 8.5l.036-.035L.5 1.428 1.928 0l7.036 7.036L15.999 0z" id="a" /></defs>
+    <use fill="#121212" xlinkHref="#a" fillRule="evenodd" />
+  </svg>);
```

Comment:
Not a blocker on this PR, but I believe we've enabled `react-svg-loader` in the webpack configuration which means we should be able to store `svg` images as `.svg` files and import them as components, eg: `import Arrow from './arrow.svg'`.

Reply:
Thanks George. I've not been able to make that work, but I am going to make it a shared arrow between the two components that use that arrow.